ipaserver/install/krbinstance.py

```diff
@@ -502,10 +502,16 @@ def issue_ipa_ca_signed_pkinit_certs(self):
             self._install_pkinit_ca_bundle()
             self.pkinit_enable()
         except RuntimeError as e:
-            logger.error("PKINIT certificate request failed: %s", e)
-            logger.error("Failed to configure PKINIT")
+            logger.warning("PKINIT certificate request failed: %s", e)
+            logger.warning("Failed to configure PKINIT")
+
+            self.print_msg("Full PKINIT configuration did not succeed")
+
             self.stop_tracking_certs()
             self.issue_selfsigned_pkinit_certs()
+            self.print_msg(
```

Comment:
The feature will not be functional after this fallback. Could we maybe instead of this add a message "The setup will only install bits essential to the server functionality" after the "Full PKINIT configuration did not succeed". You can also add a hint on how to enable PKINIT after the setup completed.
The reason for this is that with the current message, I still get a feeling that PKINIT is available at least locally for me (whatever someone might think that means).

Reply:
Thanks!